.github/dependabot.yml

```
@@ -4,19 +4,25 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+# Disable version updates; we will get them when we update from upstream Prometheus.
+    open-pull-requests-limit: 0
```

[Nit] Aligned indentation for readability:
```suggestion
    # Disable version updates; we will get them when we update from upstream Prometheus.
    open-pull-requests-limit: 0
```